composer.json

```diff
@@ -13,16 +13,21 @@
     "require": {
         "php": ">=5.3.2",
```

Comment:
Must be bumped to `>=5.5` if you use short array syntax.

BTW the better way is `^5.5 || ^7.0`.